packages/components/src/TimeFormatter/TimeFormatter.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+interface TimeFormatterProps {
+  /**
+   * Civil Time string to be formatted
+   * 17:45:00.000000000
+   * 11:30:00.000000000
+   */
+  readonly civilTime: string;
+
+  /**
+   * Decide to show 24 hour vs.
+   * 12 hour display
+   */
+  readonly useMilitaryTime: boolean;
+}
+
+export function TimeFormatter(props: TimeFormatterProps) {
+  const timeSplit = props.civilTime.split(":");
+  const hour = parseInt(timeSplit[0]);
+  const minute = parseInt(timeSplit[1]);
+
+  const amOrPm = getAmPmLabel(hour, props.useMilitaryTime);
+  const formattedHour = setCorrectHourFormat(hour, props.useMilitaryTime);
+
+  return `${formattedHour}:${minute} ${amOrPm}`;
```

Comment:
@kkimble006 Do you think there's any value in error checking any of these values (e.g. hours not between 0-23 and minutes/seconds not between 0-59)? Or should we just trust that whoever is consuming this component does so responsibly?

Reply:
I feel like since this component is accepting a CivilTime we should rely on it to ensure that it is a valid time…

Reply:
We are probably also best using the browsers built in locale formatters here. They are going to deal with a lot more edge cases so we don't have to. :)

eg
```ts
(new Date('August 19, 1975 23:15:00 GMT+00:00')).toLocaleTimeString(undefined, {timeStyle: "short"})
```